specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,45 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/backupStorageRedundancies": {
```

Comment:
Was this GET api discussed in RP API Review ?

Reply:
These URLs represent hierarchical resources. This URL means there is a resource called backupStorageRedundancy under the resource Location. This is not correct. 
This violates both REST API and ARM guidelines. Please re-design this API. Is this API really required? Correct me if I am wrong, this API tells the available values for a region.

Reply:
We do have hierarchical resources here. For each location resource, it has it's own set of backupStorageRedundancy.
As for this URL, we once confirmed with Wenbin and Devendra. Initially plan is to expose it to portal, now we also want take this chance to expose it in cli and powershell

Reply:
Looks like Azure storage does similarly https://docs.microsoft.com/en-us/rest/api/storagerp/skus/list
We should be good too.